launchable/__main__.py

```diff
@@ -1,16 +1,28 @@
 from .version import __version__
 import click
 import importlib
+import logging
 from os.path import dirname, basename, join
 from glob import glob
 from .commands.record import record
 from .commands.subset import subset
 from .commands.verify import verify
+from .utils import logger
 
 
 @click.group()
 @click.version_option(version=__version__, prog_name='launchable-cli')
-def main():
+@click.option(
+    '--log-level',
+    'log_level',
+    help='Set logger\'s log level (CRITICAL, ERROR, WARNING, AUDIT, INFO, DEBUG).',
+    type=str,
+    default=logger.LOG_LEVEL_DEFAULT_STR,
+)
+def main(log_level):
+    level = logger.get_log_level(log_level)
+    logging.basicConfig(level=level)
```

Comment:
@Konboi  I found this line affects test failure! This is because this line runs to set the config globally before `test_logger.py` tests run.

Reply:
Ohhh, Thank you so much!!! I'll try to avoid this method.